rpm/Makefile

```diff
@@ -0,0 +1,100 @@
+# Definitions for making an RPM, cribbed from Pbench Server
+#
+# Making a pbench component RPM requires a few steps:
+# 1. Get version number.
+# 2. Update the RPM spec file with that version number etc.
+# 3. Update any GIT submodules.
+# 4. Do a "make install" to a temp directory.
+# 5. Generate a tar ball from the directory.
+# 6. Generate a local SRPM that will be uploaded to COPR for building.
+# 7. Optionally generate a local RPM.
+# 8. Clean up the temp directory
+
+# NOTE:
+# Currently, we have a static spec file instead of generating one from a
+# template, so the version number, etc. is not dynamically generated.
```

Comment:
Sure you don't want to fix this before it goes in? 😁

Reply:
I think this is a "perfect is the enemy of the done" thing -- once we decide that we're going to carry the existing Dashboard forward, perhaps I'll be more enthusiastic about addressing this.  🙂

Reply:
FWIW, this functionality is provided for by `rpkg-util` which COPR uses to build the RPM, so we should now be covered.  And, given that I've dropped this file from the PR, I think it's now moot.